tests/test_invoice_playwright.py

```diff
@@ -0,0 +1,73 @@
+"""Playwright tests for invoice generator."""
+
+import os
+import subprocess
+from pathlib import Path
+
+from playwright.sync_api import Page
+
+
+def test_invoice_generation_and_content(page: Page):
+    """Test that invoice generation creates HTML file with expected content."""
+
+    # Get paths to test data files
+    test_dir = Path(__file__).parent
+    client_file = test_dir / "test-client.json"
+    invoice_data_file = test_dir / "invoice-data-3-15.txt"
+
+    # Run invoice generation in the project root directory
+    original_cwd = os.getcwd()
+
+    # Run invoice generation
+    result = subprocess.run(
```

Comment:
This will be common across lots of tests. Is there a good way to extract running this somewhere while keeping inline with python best practices?